e3sm_diags/driver/zonal_mean_2d_driver.py

```diff
@@ -187,7 +196,8 @@ def run_diag(parameter):
                     mv1_reg = mv1_p
                     mv2_reg = mv2_p
```

Comment:
Hi @chengzhuzhang , thanks for fixing this. The calculation procedure looks good and the new results are reasonable.

Don't know what is the purpose of computing mv1_p and mv2_p and the assignment to mv#_reg now. Are they still needed?